spec/spec_helper.rb

```diff
@@ -75,3 +91,19 @@ def in_tmp_dir
     end
   end
 end
+
+private
+
+def stringify_for_fastlane(hash)
+  hash.map do |key, value|
+    # rubocop:disable Style/CaseLikeIf
+    if value.is_a?(Hash)
```

Comment:
This fails to cover the case of `Array` as well. But it happens that some of our actions expect arrays of strings or arrays of hashes as parameters.

For example `an_localize_libs_action` has its `libs_strings_path` parameter which is expected to be an Array of Hashes…

Reply:
Oh, and also even if we don't _yet_ have any case like that currently, iirc we talked about the idea of starting to sometimes use small DTOs as parameters for some actions that currently require a convoluted and less readable Hash structure for some of their parameters (`an_localize_libs_action` comes to mind, but I believe the one we were talking about back then was for cases like the Hash of the locales map (see [this draft](https://github.com/wordpress-mobile/release-toolkit/pull/296/files#diff-4d1e645f1d455e17026d35880050ced2a9f0201d00ef958024c9a1108bf25a85R30)) for which we might consider in the future that some of our actions could accept `[Locale]` instances instead of `[Hash]` for readability in our Fastfiles… though for that one, I don't think there would be a magic solution to stringify those cases for us 😒  but just thought I'd mention that case to provide some water to the mill of brainstorming solutions for that. Tho in the end we might consider just not supporting those special cases… 🤷